src/main/java/seedu/address/commons/util/AppUtil.java

```diff
@@ -39,4 +41,21 @@ public static void checkArgument(Boolean condition, String errorMessage) {
             throw new IllegalArgumentException(errorMessage);
         }
     }
+
+    /**
+     * Checks that lists {@code first} and {@code second} have the same number of elements and that
+     * each element in index i in {@code first} is equal (based on the .equals() function) to the element
+     * in index i in {@code second}.
+     */
+    public static <T> boolean areListsEqual(List<T> first, List<T> second) {
```

Comment:
What is this equality used for? Looks like it only allows for lists whose elements are in the same order. Not sure if that's the expected behaviour you're going for.